src/test/java/com/google/api/codegen/testdata/java/java_library.baseline

```diff
@@ -4096,35 +4097,13 @@ import java.util.Arrays;
 import java.util.List;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import org.apache.commons.cli.CommandLine;
-import org.apache.commons.cli.DefaultParser;
-import org.apache.commons.cli.HelpFormatter;
-import org.apache.commons.cli.Option;
-import org.apache.commons.cli.Options;
-import org.apache.commons.lang.builder.ReflectionToStringBuilder;
-import org.apache.commons.lang.builder.ToStringStyle;
 
 @javax.annotation.Generated("by GAPIC")
 public class SampleApp {
   public static void main(String args[]) {
     Logger.getLogger("").setLevel(Level.WARNING);
     try {
-      Options options = new Options();
-      options.addOption("h", "help", false, "show usage");
-      options.addOption(
-          Option.builder()
-              .longOpt("project_id")
-              .desc("Project id")
-              .hasArg()
-              .argName("PROJECT-ID")
-              .required(true)
-              .build());
-      CommandLine cl = (new DefaultParser()).parse(options, args);
-      if (cl.hasOption("help")) {
-        HelpFormatter formater = new HelpFormatter();
-        formater.printHelp("SampleApp", options);
-      }
-      executeNoCatch(cl.getOptionValue("project_id"));
+      executeNoCatch(getProjectId());
```

Comment:
I think I still want to allow the user to pass in a project id on the command line for clients where we don't automatically look up the project id.

Reply:
With this setup, user can pass project id through sys prop and env var. Do you mean we still need a command line interface, treating smoke test as an app ?

Reply:
It's nice that we can configure through an environment variable, but I kind of wanted to retain the ability to pass in things like project id on the command line.